add-on/chrome/modules/MozLoopAPI.jsm

```diff
@@ -192,6 +192,16 @@ const kMessageHandlers = {
   },
 
   /**
+   * Triggered at ROOM_STATES.SESSION_CONNECTED meant to notify the connection
+   * status.
+   */
+  RoomSessionConnected: function(message, reply) {
+    let win = Services.wm.getMostRecentWindow("navigator:browser");
+    win.LoopUI.roomSessionConnected();
```

Comment:
We should probably have a better way to make sure we call `roomSessionConnected` to the correct window, but I see this pattern elsewhere in this file, so this is fine for now.